src/lib/components/FormFeedback.js

```diff
@@ -35,14 +34,15 @@ const defaultLabels = {
   'metadata.version': 'Version',
   'metadata.publisher': 'Publisher',
   'metadata.related_identifiers': 'Related works',
-}
+  'access.embargo.until': 'Embargo until',
```

Comment:
We are slightly breaking the pattern of only showing the top level label here (other fields are not as specific)... which is fine for now (we shouldn't really care about perfect consistency). But we should keep an eye out to see if that devolved over the longer term.